updated the geo lat,long with high accuracy codes

charanrajtc opened the pull request:
## ISSUE 

- Distance b/w zip 77059  to 77586 was showing 26 miles because of lat long precision.
actually it is around 5.33 miles https://www.searchbug.com/tools/zip-radius.aspx?TYPE=zipradius&ZIP=77059&DIST=20&submit=Search

## FIX
- updated the lat, long data form http://geonames.org to provide high accuracy distance computing. 

-----------------
Signed-off-by: Raj <charanraj.tc@gmail.com>

davglass commented on scripts/fetch.sh:
```diff
@@ -5,17 +5,29 @@ set -e
     $wget -nv "http://federalgovernmentzipcodes.us/free-zipcode-database.csv"
 fi
 
+if [ ! -f ./US.t ]; then
```
Shouldn't this be:

```shell
if [ ! -f ./US.txt ]; then
```

davglass commented on scripts/process.js:
```diff
@@ -2,12 +2,13 @@
+    zips = {},
+    str,
     data = fs.readFileSync('./free-zipcode-database.csv', 'utf8').replace(/\r/g, '').split('\n');
+data2 = fs.readFileSync('./US.txt', 'utf8').split('\n');
```
This should be:

```js
    data = fs.readFileSync('./free-zipcode-database.csv', 'utf8').replace(/\r/g, '').split('\n'),
    data2 = fs.readFileSync('./US.txt', 'utf8').split('\n');
```

davglass commented on scripts/process.js:
```diff
@@ -55,4 +73,4 @@ for (var i in zips) {
 str += 'exports.stateMap = ' + JSON.stringify(stateMap) + ';\n';
 
-fs.writeFileSync(path.join('../', 'lib', 'codes.js'), str, 'utf8');
+fs.writeFileSync(path.join('../', 'lib', 'codes.js'), str, 'utf8');
```
Please add a linefeed back here.

charanrajtc commented:
done

Cheers,

{
  "name": "*Charan Raj T C*",
  "email": "charanraj.tc@gmail.com <mailto:charanraj.tc@gmail.com>",
  "telephone": "+91 96866 98989 <+919686698989>",
  "url": "http://coresoft.co.in/charanraj.tc"
}                            	


On 13 October 2017 at 21:07, Dav Glass <notifications@github.com> wrote:

> [image: Boxbe] <https://www.boxbe.com/overview> Dav Glass (
> notifications@github.com) is on your Guest List
> <https://www.boxbe.com/approved-list?tc_serial=33490193931&tc_rand=988336183&utm_source=stf&utm_medium=email&utm_campaign=ANNO_SELF&utm_content=001&key=n%2FD%2FkMDL0f%2F%2FrrWZhtvAcrYE39C5%2BQkipWLlo18nrr0%3D&token=7Xa69C0DAg1aFHTtWzIpI3YuXPKBIDZuKeAPU8sIUXeY%2B6ipIrfHoUzSYQOn3aAB>
> | Delete this guest
> <https://www.boxbe.com/anno?action=remove&tc_serial=33490193931&tc_rand=988336183&utm_source=stf&utm_medium=email&utm_campaign=ANNO_SELF&utm_content=001&key=n%2FD%2FkMDL0f%2F%2FrrWZhtvAcrYE39C5%2BQkipWLlo18nrr0%3D&token=7Xa69C0DAg1aFHTtWzIpI3YuXPKBIDZuKeAPU8sIUXeY%2B6ipIrfHoUzSYQOn3aAB>
>
> *@davglass* requested changes on this pull request.
>
> Thanks for this! Just a few comments on it. Can you also please remove the
> package-lock.json file? It shouldn't be in a module repo.
> ------------------------------
>
> In scripts/fetch.sh
> <https://github.com/davglass/zipcodes/pull/17#discussion_r144587191>:
>
> >  if [ ! -f ./free-zipcode-database.csv ]; then
>      echo "Fetching US Zipcodes CSV File"
>      $wget -nv "http://federalgovernmentzipcodes.us/free-zipcode-database.csv"
>  fi
>
> +if [ ! -f ./US.t ]; then
>
> Shouldn't this be:
>
> if [ ! -f ./US.txt ]; then
>
> ------------------------------
>
> In scripts/process.js
> <https://github.com/davglass/zipcodes/pull/17#discussion_r144587457>:
>
> >      data = fs.readFileSync('./free-zipcode-database.csv', 'utf8').replace(/\r/g, '').split('\n');
> +data2 = fs.readFileSync('./US.txt', 'utf8').split('\n');
>
> This should be:
>
>     data = fs.readFileSync('./free-zipcode-database.csv', 'utf8').replace(/\r/g, '').split('\n'),
>     data2 = fs.readFileSync('./US.txt', 'utf8').split('\n');
>
> ------------------------------
>
> In scripts/process.js
> <https://github.com/davglass/zipcodes/pull/17#discussion_r144587640>:
>
> > @@ -55,4 +73,4 @@ for (var i in zips) {
>  str = 'exports.codes = ' + JSON.stringify(zips) + ';\n';
>  str += 'exports.stateMap = ' + JSON.stringify(stateMap) + ';\n';
>
> -fs.writeFileSync(path.join('../', 'lib', 'codes.js'), str, 'utf8');
> +fs.writeFileSync(path.join('../', 'lib', 'codes.js'), str, 'utf8');
>
> Please add a linefeed back here.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/davglass/zipcodes/pull/17#pullrequestreview-69267271>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACCpFav4aX0CeOcxFpE1UtPAgvTVkNXuks5sr4O5gaJpZM4P4dfM>
> .
>
>

davglass commented:
Published in `zipcodes@5.0.0`
